Spec/Auth.swift

```diff
@@ -558,6 +558,18 @@ class Auth : QuickSpec {
                         }
                     }
 
+                    // RSA7b4
+                    it("client does not have an identity when a wildcard string '*' is present") {
+                        let realtime = ARTRealtime(options: AblyTests.commonAppSetup())
```

Comment:
This is not clear, we need to demonstrate `When a wildcard string '*' is present in the TokenRequest, TokenDetails, or ProtocolMessage#connectionDetails object` in the test.  So whilst it may in fact be in the `connectionDetails`, I would prefer to be more explicit here and use a token with a wildcard